protobuf/mayastor.proto

```diff
@@ -96,13 +96,6 @@ message Null {}
 message CreatePoolRequest {
   string name = 1;           // name of the pool
   repeated string disks = 2; // disk device paths or URIs to be claimed by the pool
-  PoolType pooltype = 3; // type of the pool
-}
-
```

Comment:
These changes have been already reverted in 38e217e841f796b8bc9f95d82e3d08742d4b7af0

Reply:
:+1: